OrbitGl/App.cpp

```diff
@@ -1124,8 +1124,10 @@ void OrbitApp::StartCapture()
 #ifdef WIN32
     Capture::StartCapture();
 #else
-    if( Capture::StartCapture() )
+    if( Capture::StartCapture() && !Capture::IsRemote() )
```

Comment:
Should we check !Capture::IsRemote first so to not call StartCapture in the case whre IsRemote is true?

Reply:
I think that StartCapture here is still necessary even if Capture::IsRemote. The problem is that the BpfTrace should not be here but I don't want to refactor this in this change as we're going to remove BpfTrace as soon as possible, so this was just a quick fix.